src/main/java/org/terasology/climateConditions/VisibleBreathComponent.java

```diff
@@ -3,6 +3,10 @@
 import org.terasology.entitySystem.Component;
 import org.terasology.entitySystem.entity.EntityRef;
 
+/**
+ * Augments the world with a visible breath particle effect.
```

Comment:
"Augments the world" sounds a bit ... broad and unspecific. I'd be a bit more specific and explain that this only has the desired effect if the entity also has a `LocationComponent` that determines the location and direction of the effect.